packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -193,8 +197,16 @@ export const usePopover = (options: PopoverOptions & PopoverInternalProps) => {
 // ----------------------------------------------------------------------------
 
 type PopoverPublicProps = {
+  /**
+   * Content displayed inside the popover.
+   */
   content?: React.ReactNode;
   children?: React.ReactNode;
```

Comment:
Maybe we can add JSDocs to say this is the trigger?

Reply:
the `children` jsdocs don't get exposed anywhere